target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -126,6 +126,10 @@ ath79_setup_interfaces()
 	ubnt,routerstation)
 		ucidef_set_interfaces_lan_wan "eth1" "eth0"
 		;;
+	comfast,cf-e375ac)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "1:wan" "2:lan"
+		;;
```

Comment:
Same MAC address for lan and wan?

Reply:
It was defined as this for the ar71xx target, so I kept the definition initially.
According to the original firmware, the wan interface mac is  lan MAC +1 in /etc/config/network.

Reply:
I'm still not sure how to patch the wan mac address. 
An entry in ath79_setup_macs will work?

Reply:
Yes.